services/gundeck/src/Gundeck/Push.hs

```diff
@@ -229,7 +229,7 @@ pushFallback orig notif r prio = case Fallback.prepare orig r of
     Just can ->
         if ntfTransient notif
             then Log.warn $ msg (val "Transient notification failed")
-            else void $ Fallback.execute (ntfId notif) prio can
+            else void $ Fallback.execute (ntfId notif) (ntfTransient notif) prio can
```

Comment:
In this case, `ntfTransient notif` is always `False` (see `if` condition two lines above).

Reply:
third possibility:

3. the execution paths are carefully or luckily engineered such that no `Notice` constructed from a transient `Notification` is every actually pushed natively.